assets/src/edit-story/components/checklist/constants.js

```diff
@@ -469,6 +470,29 @@ export const PRIORITY_COPY = {
       `${MAX_VIDEO_RESOLUTION}p`
     ),
   },
+  storyMissingPublisherName: {
+    title: __('Add Site Title', 'web-stories'),
+    footer: (
+      <TranslateWithMarkup
+        mapping={{
+          a: (
+            <Link
+              href={__('/wp-admin/options-general.php', 'web-stories')}
```

Comment:
is this translatable?

Reply:
No.

But also please don't put hardcoded links like that in JS.

Define the URL in PHP here to pass it to the config provider:

https://github.com/google/web-stories-wp/blob/6f4b205b065ed84516557d8f6d53d22cb70e33f6/includes/Admin/Editor.php#L252-L293

Also, check whether the user has permission to edit settings first. Otherwise they cannot access the settings page.

```php

$general_settings_url = admin_url( 'options-general.php' );

// ...

'generalSettingsLink' => $general_settings_url,

// ...
'capabilities' => [
  'canManageSettings'   => current_user_can( 'manage_options' ),

```

Reply:
thank youuuu, on it

Reply:
should be all better, much appreciated details, thanks.